library/psa_util.c

```diff
@@ -28,9 +28,12 @@
 #include <mbedtls/psa_util.h>
 #include <mbedtls/error.h>
 #include <mbedtls/lms.h>
-#include <mbedtls/ssl.h>
 #include <mbedtls/rsa.h>
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO) || defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
Why do we need `ssl.h` if `USE_PSA_CRYPTO` is defined? Surely we might be doing crypto only, not TLS?

Reply:
Change removed as part of the rebase as it was addressed by abfe640864fc041e16f09356cf049a3c113532fa.